src/scripts/farming/Farming.ts

```diff
@@ -1083,7 +1082,11 @@ class Farming implements Feature {
         }
 
         amount = Math.min(this.mulchList[mulch](), amount);
+
         GameHelper.incrementObservable(this.mulchList[mulch], -amount);
+        GameHelper.incrementObservable(App.game.statistics.totalMulchesUsed, amount);
+        GameHelper.incrementObservable(App.game.statistics.mulchesUsed[+mulch], amount);
```

Comment:
We shouldn't need to use `+mulch` anymore, as it should always be a number due to the ko extend.
```suggestion
        GameHelper.incrementObservable(this.mulchList[mulch], -amount);
        GameHelper.incrementObservable(App.game.statistics.totalMulchesUsed, amount);
        GameHelper.incrementObservable(App.game.statistics.mulchesUsed[mulch], amount);
```